### PR TITLE
Rework all code fixed tests.

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ✔️
| Tests pass?   | ✔️

Rework code fixed tests:
- [x] ActiveQueryTest.
- [x] ActiveRecordTest.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -42,3 +42,4 @@ phpunit.phar
 # NPM packages
 /node_modules
 .env
+composer.json
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply .gitignore]: It's strange but I already corrected it.

**Comment by samdark** [comment]
Is it overall ready for review?

**Comment by terabytesoftw** [comment]
The tests only fail due to the change of operation of `ArrayHelper::getValue()`, I must check and adjust the code so that it does not fail.

**Comment by samdark** [comment]
Let's have it in. We can do more cleanup later when tests are in place and are green.

**Pull request merged**